### PR TITLE
doc: Add links of the nRF9161 Hardware and PS guides in the NCS docs

### PR DESCRIPTION
**Pull request by divipillai**
Add links of Hardware and PS docs
to NCS.
NCSDK-23571

### CODE REVIEW

**Review by MarkusLassila — APPROVED** [review]

**Review by MarkusLassila — APPROVED** [review]

**Review by juhaylinen — APPROVED** [review]

**Review by SeppoTakalo — APPROVED** [review]

File: doc/nrf/security/ap_protect.rst [hunk 1]
```diff
@@ -59,6 +59,11 @@ See the related hardware documentation for more information about which implemen
      - Hardware and software AP-Protect support
      - Related hardware documentation
      - Additional information
+   * - nRF9161
+     - ✔
+     -
+     - `Access protection for nRF9161`_
+     - Also supports Secure AP-Protect (see note below)
```
Comment (greg-fer) [inline doc/nrf/security/ap_protect.rst:66]: @anangl , can you please confirm this information is correct for nRF9161? Thanks.

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: doc/nrf/links.txt [hunk 2]
```diff
@@ -533,30 +532,46 @@
...
+.. _`nRF91x1 AT+CNEC set command`: https://infocenter.nordicsemi.com/topic/ref_at_commands_nrf91x1/REF/at_commands/mobile_termination_errors/cnec_set.html
+.. _`nRF91x1 Power saving mode setting`: https://infocenter.nordicsemi.com/topic/ref_at_commands_nrf91x1/REF/at_commands/nw_service/cpsms.html
+.. _`nRF91x1 modem trace activation %XMODEMTRACE`: https://infocenter.nordicsemi.com/topic/ref_at_commands_nrf91x1/REF/at_commands/mob_termination_ctrl_status/xmodemtrace.html
 
 .. _`nRF5340 Product Specification`: https://infocenter.nordicsemi.com/topic/ps_nrf5340/keyfeatures_html5.html
 .. _`nRF5340 DK User Guide`: https://infocenter.nordicsemi.com/topic/ug_nrf5340_dk/UG/dk/intro.html
 .. _`Execute in place page in the nRF5340 Product Specification`: https://infocenter.nordicsemi.com/topic/ps_nrf5340/qspi.html#execute_in_place
 .. _`nRF5340 DK Compatibility Matrix`: https://infocenter.nordicsemi.com/topic/comp_matrix_nrf5340/COMP/nrf5340/nrf5340_comp_matrix.html
+.. _`Debugging nRF5340 with SES`: https://infocenter.nordicsemi.com/topic/ug_gsg_ncs/UG/gsg/debug.html?cp=1_1_0_7
```
[the remark this answers is not in the document]
Comment (divipillai) [reply doc/nrf/links.txt:574]: Yeah, will remove the link.

File: doc/nrf/security/ap_protect.rst (hunk 1 above)
Comment (divipillai) [reply doc/nrf/security/ap_protect.rst:66]: Please note this was confirmed by @frkv

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst [hunk 3]
```diff
@@ -172,7 +172,7 @@ The band lock mask allows you to set the bands on which you want the modem to op
 Each bit in the :kconfig:option:`CONFIG_LTE_LOCK_BAND_MASK` option represents one band.
 The maximum length of the string is 88 characters (bit string, 88 bits).
 
-For more detailed information, see the `band lock section in the AT Commands reference document`_.
+For more detailed information, see the `band lock section in the nRF9160 AT Commands Reference Guide`_ or `band lock section in the nRF91x1 AT Commands Reference Guide`_, depending on the SiP you are using.
```
Comment (umapraseeda) [inline doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst:175]:
```suggestion
For more detailed information, see the `band lock section in the nRF9160 AT Commands Reference Guide`_ or the `band lock section in the nRF91x1 AT Commands Reference Guide`_, depending on the SiP you are using.
```

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst [hunk 4]
```diff
@@ -188,7 +188,7 @@ You can change the enabled systems using the :kconfig:option:`CONFIG_LTE_NETWORK
 When the :ref:`lte_lc_readme` library is not used, the modem starts in LTE-M mode.
 You can change the system mode and the LTE system mode preference using the ``AT%XSYSTEMMODE`` AT command.
 
-For more detailed information, see the `system mode section in the AT Commands reference document`_.
+For more detailed information, see the `system mode section in the nRF9160 AT Commands Reference Guide`_ or `system mode section in the nRF91x1 AT Commands Reference Guide`_, depending on the SiP you are using.
```
Comment (umapraseeda) [inline doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst:191]:
```suggestion
For more detailed information, see the `system mode section in the nRF9160 AT Commands Reference Guide`_ or the `system mode section in the nRF91x1 AT Commands Reference Guide`_, depending on the SiP you are using.
```

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst [hunk 5]
```diff
@@ -212,7 +212,7 @@ To enable the modem traces in the modem and to forward them to the :ref:`modem_t
    For the :ref:`serial_lte_modem` application and the :ref:`at_client_sample` sample, you must also run ``AT%XMODEMTRACE=1,2`` to manually activate the predefined trace set.
 
 You can set the trace level using the AT command ``AT%XMODEMTRACE``.
-See `modem trace AT command documentation`_ for more information.
+For more information, see `modem trace activation %XMODEMTRACE`_ section in the nRF9160 AT Commands Reference Guide or the `same section <nRF91x1 modem trace activation %XMODEMTRACE_>`_ in the nRF91x1 AT Commands Reference Guide, depending on the SiP you are using.
```
Comment (umapraseeda) [inline doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst:215]:
```suggestion
For more information, see the `modem trace activation %XMODEMTRACE`_ section in the nRF9160 AT Commands Reference Guide or the `same section <nRF91x1 modem trace activation %XMODEMTRACE_>`_ in the nRF91x1 AT Commands Reference Guide, depending on the SiP you are using.
```

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst [hunk 6]
```diff
@@ -351,7 +351,7 @@ Concurrent GNSS and LTE
 
 The GNSS operation in an nRF91 Series device is time-multiplexed with the LTE modem.
 Therefore, the LTE modem must either be completely deactivated or in `RRC idle mode <Radio Resource Control_>`_ or `Power Saving Mode (PSM)`_ when using the GNSS receiver.
-For an nRF9160 SiP, see the `nRF9160 GPS receiver Specification`_ for more information.
+For more information, see the `nRF9161 GPS receiver Specification`_ or `nRF9160 GPS receiver Specification`_, depending on the SiP you are using.
```
Comment (umapraseeda) [inline doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst:354]:
```suggestion
For more information, see the `nRF9161 GPS receiver Specification`_ or the `nRF9160 GPS receiver Specification`_, depending on the SiP you are using.
```

File: samples/cellular/pdn/README.rst [hunk 7]
```diff
@@ -30,7 +30,7 @@ Finally, the sample prints the PDP context IDs and PDN IDs of both the default P
 .. note::
    The sample uses the :ref:`lte_lc_readme` library to change the modem's functional mode.
    Hence, the :ref:`pdn_readme` library can automatically register to the necessary packet domain events notifications using the ``AT+CGEREP=1`` AT command, and notifications for unsolicited reporting of error codes sent by the network using the ``AT+CNEC=16`` AT command.
-   See the `AT+CGEREP set command`_  and the `AT+CNEC set command` sections, respectively, in the nRF9160 AT Commands Reference Guide or the same sections in the `nRF91x1 AT Commands Reference Guide`_ depending on the SiP you are using.
+   See the `AT+CGEREP set command <AT+CGEREP set command_>`_  and the `AT+CNEC set command <AT+CNEC set command_>`_ sections, respectively, in the nRF9160 AT Commands Reference Guide or the `AT+CGEREP set command <nRF91x1 AT+CGEREP set command_>`_  and the `AT+CNEC set command <nRF91x1 AT+CNEC set command_>`_ sections in the nRF91x1 AT Commands Reference Guide, depending on the SiP you are using.
```
Comment (umapraseeda) [inline samples/cellular/pdn/README.rst:33]:
```suggestion
   See the `AT+CGEREP set command <AT+CGEREP set command_>`_  and the `AT+CNEC set command <AT+CNEC set command_>`_ sections, respectively, in the nRF9160 AT Commands Reference Guide or the `AT+CGEREP set command <nRF91x1 AT+CGEREP set command_>`_ and the `AT+CNEC set command <nRF91x1 AT+CNEC set command_>`_ sections in the nRF91x1 AT Commands Reference Guide, depending on the SiP you are using.
```

**Review by umapraseeda — CHANGES_REQUESTED** [review]

**Review by lemrey — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

File: doc/nrf/links.txt [hunk 8]
```diff
@@ -616,6 +630,7 @@
 .. _`nPM1300`: https://infocenter.nordicsemi.com/topic/struct_pmic/struct/npm1300.html
 .. _`nPM1300 EK User Guide`: https://infocenter.nordicsemi.com/topic/ug_npm1300_ek/UG/nPM1300_EK/intro.html
 
+.. _`Access protection for nRF9161`: https://infocenter.nordicsemi.com/index.jsp?topic=%2Fps_nrf9161%2Fdif.html&anchor=ariaid-title3
```
Comment (greg-fer) [inline doc/nrf/links.txt:633]: Please match the formatting of other similar links in the table, given that the section name is the same as in their cases:
```suggestion
.. _`AP-Protect for nRF9161`: https://infocenter.nordicsemi.com/index.jsp?topic=%2Fps_nrf9161%2Fdif.html&anchor=ariaid-title3
```

File: doc/nrf/security/ap_protect.rst [hunk 9]
```diff
@@ -59,6 +59,11 @@ See the related hardware documentation for more information about which implemen
      - Hardware and software AP-Protect support
      - Related hardware documentation
      - Additional information
+   * - nRF9161
+     - ✔
+     -
+     - `Access protection for nRF9161`_
```
Comment (greg-fer) [inline doc/nrf/security/ap_protect.rst:65]:
```suggestion
     - `AP-Protect for nRF9161`_
```

File: doc/nrf/security/ap_protect.rst (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/security/ap_protect.rst:66]: You mean confirmed that the procedures for enabling AP-Protect for nRF9161 are matching the contents of the sections below this table?

I'd like @anangl to still have a look, as he co-authored this page with Simen back in the days.

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: doc/nrf/security/ap_protect.rst (hunk 1 above)
Comment (divipillai) [reply doc/nrf/security/ap_protect.rst]: I meant that it is confirmed that both 9160 and 9161 have support for APPROTECT and SECUREAPPROTECT. 

Sure, we can have Andrzej's input also in this.

**Review by anangl — CHANGES_REQUESTED** [review]

File: doc/nrf/security/ap_protect.rst (hunk 1 above)
Comment (anangl) [reply doc/nrf/security/ap_protect.rst]: Yes, both have APPROTECT and SECUREAPPROTECT, but the AP-Protect implementation type is different.

Comment (greg-fer) [reply doc/nrf/security/ap_protect.rst]: @anangl does this mean the procedures for enabling AP-Protect https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/security/ap_protect.html#configuration-overview-in-the-ncs are _not_ applicable to nRF9161?

Comment (anangl) [reply doc/nrf/security/ap_protect.rst]: No, the procedures are applicable. Only the type of AP-Protect implementation indicated by that checkmark above is specified incorrectly. As I also mentioned in https://github.com/nrfconnect/sdk-nrf/pull/13744#discussion_r1476132189, it should be "Hardware and software" for nRF9161,

**Review by anangl — APPROVED** [review]

File: doc/nrf/security/ap_protect.rst (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/security/ap_protect.rst]: Thanks, @anangl !

**Review by greg-fer — APPROVED** [review]
Approving changes to ap_protect.rst and links.txt.

**Review by umapraseeda — APPROVED** [review]

**Review by frkv — APPROVED** [review]
LGTM

**Pull request merged**